<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomUI</class>
 <widget class="QWidget" name="WelcomUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WelcomUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_logo">
        <property name="font">
         <font>
          <family>Adobe 楷体 Std R</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>华清远见智能售货机</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,3">
        <item>
         <widget class="VideoADUI" name="widget_Video" native="true"/>
        </item>
        <item>
         <widget class="AdvertisementUI" name="widget_ADPicture" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="AdvertisementUI" name="widget_ADText" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <zorder></zorder>
       </widget>
      </item>
     </layout>
     <zorder>widget_ADText</zorder>
     <zorder>label_logo</zorder>
     <zorder>widget_ADText</zorder>
     <zorder></zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AdvertisementUI</class>
   <extends>QWidget</extends>
   <header>advertisementui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoADUI</class>
   <extends>QWidget</extends>
   <header>videoadui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
